tools/debug_tool/templates/pipeline_tree.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+<meta charset="utf-8" />
+<title>Downward Edges</title>
+<style>
+
+
+.node {
+  fill: rgba(32, 202, 117, 0.452);
+  /* stroke: #fff; */
+  /* stroke-width: 1.5px; */
+}
+
+.link {
+  fill: none;
+  stroke: #000;
+  stroke-width: 1.5px;
+  opacity: 0.4;
+  marker-end: url(#end-arrow);
+}
+
+.label {
+  fill: rgb(0, 0, 0);
+  font-family: sans-serif;
+  font-size: 15;
+}
+
+</style>
+</head>
+<body>
+    <a href="../index.html">cola.js home</a>
+<h1>Kotekan Pipeline</h1>
+<script src="{{url_for('static', filename='d3v4.js')}}"></script>
+<script src="{{url_for('static', filename='cola.min.js')}}"></script>
+<script type="text/javascript" src="https://canvasjs.com/assets/script/jquery-1.11.1.min.js"></script>
+<script>
+    var width = 960,
+        height = 500;
+  //var w = document.getElementById("graph").clientWidth;
+  //var h = document.getElementById("graph").clientHeight;
+
+    var color = d3.scaleOrdinal(d3.schemeCategory20);
+
+    var d3cola = cola.d3adaptor(d3)
+        //.handleDisconnected(false)
+        .linkDistance(80)
+        //.groupCompactness(100)
+        .size([width, height]);
+
+    var svg = d3.select("body").append("svg")
+        .attr("width", width)
+        .attr("height", height)
+        .call(d3.zoom().on("zoom", function () {
+            svg.attr("transform", d3.event.transform)
+        }));
+
+    // Replace "&#39;" chars as single quote "'"
+    var rawData =  "{{data.json()}}".split('&#39;').join('"');
+    // Add double quotes to None elements ('None') then parse json into javascript object
+    var buffers = jQuery.parseJSON(rawData.split('None').join('"None"'));
+    // Get buffer names
+    var bufNames = Object.keys(buffers);
+        
+    console.log(buffers);
+
+    d3.json("{{url_for('static', filename='chris.json')}}", function (error, graph) {
```

Comment:
So we don't actually use the `chris.json` file here. So i think that part can be removed